ansible_events_ui/api/activation.py

```diff
@@ -182,41 +160,56 @@ async def create_activation_instance(
     ),
     settings: Settings = Depends(get_settings),
 ):
-    query = sa.select(models.rulebooks).where(
-        models.rulebooks.c.id == a.rulebook_id
-    )
-    rulebook_row = (await db.execute(query)).first()
-
-    query = sa.select(models.inventories).where(
-        models.inventories.c.id == a.inventory_id
+    query = (
```

Comment:
A small bit of refactoring since I was under the hood of that function anyway. I also needed more info in the output so I don't feel this is scope-creep.